ios/KaMPKitiOS/ViewController.swift

```diff
@@ -15,12 +15,20 @@ class BreedsViewController: UIViewController {
     var data: [Breed] = []
     
     let log = koin.get(objCClass: Kermit.self, parameter: "ViewController") as! Kermit
-    
+
     lazy var adapter: NativeViewModel = NativeViewModel(
-        viewUpdate: { [weak self] summary in
-            self?.viewUpdate(for: summary)
-        }, errorUpdate: { [weak self] errorMessage in
-            self?.errorUpdate(for: errorMessage)
+        onLoading: {
+            // Show a loading spinner
+        },
+        onSuccess: { [weak self] summary in
+            self?.viewUpdateSuccess(for: summary)
+        },
+        onError: {
+            [weak self] error in
```

Comment:
This looks to me like it was just a typo, but I'd keep ```onError: { [weak self] error in ``` on a single line. It's the Swift convention and I think we also want to keep this variable declaration as succinct as possible.